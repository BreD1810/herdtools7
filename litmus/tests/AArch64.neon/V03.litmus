AArch64 V03
(* Tests LD1/ST1 single structure with 8 bit lanes *)
{
x=255;
0:X0=x; 0:X1=y;
}
 P0                 ;
 LD1 {V0.B}[15],[X0] ;
 ST1 {V0.B}[15],[X1] ;
forall(y=255)
