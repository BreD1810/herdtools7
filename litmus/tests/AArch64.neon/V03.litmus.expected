Test V03 Required
Histogram (1 states)
1000000:>y=255;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (y=255) is validated
Hash=f0481be63f9f2f91d9e08b274bb3f591
Observation V03 Always 1000000 0
